Dockerfile

```diff
@@ -48,8 +47,6 @@ WORKDIR /opt/
 
 # Import gpg key
 COPY keys/manufacturing-key.gpg ./
-RUN gpg --import manufacturing-key.gpg
-RUN rm manufacturing-key.gpg
 
 # @TODO: Re-enable health-check once Balena supports it fully.
```

Comment:
Guess we can remove these comments. They've been here for ages and we have never used

Reply:
Sure, consider it done.

Reply:
Done.

Reply:
🙏